projects/core/src/cx-api/cx-api.service.spec.ts

```diff
@@ -1,16 +1,15 @@
 import { TestBed, inject } from '@angular/core/testing';
 import { CxApiService } from './cx-api.service';
 
+import { AuthService } from '../auth';
```

Comment:
each import should use the corresponding barrels - if the barrel exists (lines 4-12)